fdbserver/storageserver.actor.cpp

```diff
@@ -8892,6 +8893,47 @@ ACTOR Future<Void> tryGetRangeForBulkLoad(PromiseStream<RangeResult> results,
 	}
 }
 
+// Utility function to process sample files during bulk load
+// Not critical if we fail to process sample files
+void processSampleFiles(StorageServer* data,
+                        const std::string& bulkLoadLocalDir,
+                        const std::shared_ptr<BulkLoadFileSetKeyMap>& localFileSets) {
+	for (const auto& [range, fileSet] : *localFileSets) {
+		if (fileSet.hasByteSampleFile()) {
+			std::string sampleFilePath = fileSet.getBytesSampleFileFullPath();
+			std::ifstream sampleFile(sampleFilePath);
+			if (sampleFile.is_open()) {
+				std::string line;
+				while (std::getline(sampleFile, line)) {
+					try {
+						// Process each line of the sample file
+						// Format is typically key\tvalue
+						size_t tabPos = line.find('\t');
```

Comment:
Can we use the sst file reader to get the byte sampled metadata? which seems to be more clean

Reply:
Done